internal/controller/config/config.go

```diff
@@ -369,7 +369,7 @@ func (config *ProgressiveConfig) GetAnalysisRunTimeout() (time.Duration, error)
 	defaultAnalysisRunTimeout := 1200
 
 	if config.AnalysisRunTimeout == "" {
-		return time.Duration(defaultAnalysisRunTimeout), nil
+		return time.Duration(defaultAnalysisRunTimeout) * time.Second, nil
```

Comment:
it looks like below on line 377 you're returning `time.Duration(defaultAnalysisRunTimeout)` - do you need to change that too?

Reply:
Yes, hadn't changed that from before since we were just only returning the error in that case. Will change that.